Scoop/Controller/Onboarding/OnboardingViewController.swift

```diff
@@ -32,11 +34,83 @@ class OnboardingViewController: UIViewController {
         view.addSubview(button)
         
         button.snp.makeConstraints { make in
-
             let screenSize = UIScreen.main.bounds
-            print(screenSize.width)
-            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(screenSize.width * trailingButtonMultiplier)
+            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(20)
             make.bottom.equalTo(view.safeAreaLayoutGuide).inset(screenSize.height * bottomButtonMultiplier)
         }
     }
+    
+    func setupBackButton() {
+        backButton.setImage(UIImage(named: "BackArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(prevVC), for: .touchUpInside)
+        self.navigationController?.navigationBar.addSubview(backButton)
+        backButton.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(25)
+            make.top.centerY.equalToSuperview()
+        }
+    }
+
+    func setBackButtonVisibility(isHidden: Bool) {
```

Comment:
You don't really need this function. You can just do `backButton.isHidden.toggle()` If a button is hidden it is automatically doesn't detect input events so isEnabled logic is not needed.